x-pack/filebeat/tests/integration/otel_test.go

```diff
@@ -0,0 +1,134 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build integration && !agentbeat
+
+package integration
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/elastic/beats/v7/libbeat/tests/integration"
+	"github.com/elastic/elastic-agent-libs/testing/estools"
```

Comment:
Estools imports elastic v8 [here](https://github.com/elastic/elastic-agent-libs/pull/265/files#diff-22ab41c3aca88456bf21440327ada0634dcf3187f9916456ae79a99cb3ed8027R33) but you import v7 here, we should probably stick with one version. AFAIK they go-elasticsearch versions are backwards compatible.

Reply:
I assume this comment is a bit misplaced and it is about the `github.com/elastic/go-elasticsearch/v7` import below.